ckanext/validation/jobs.py

```diff
@@ -117,18 +126,21 @@ def run_validation_job(resource):
          'validation_timestamp': validation.finished.isoformat()})
 
 
-def _validate_table(source, _format=u'csv', schema=None, **options):
+def _validate_table(source, _format='csv', schema=None, **options):
 
+    frictionless_context = { 'trusted': True }
     http_session = options.pop('http_session', None) or requests.Session()
 
     use_proxy = 'ckan.download_proxy' in t.config
     if use_proxy:
         proxy = t.config.get('ckan.download_proxy')
-        log.debug(u'Download resource for validation via proxy: %s', proxy)
+        log.debug('Download resource for validation via proxy: %s', proxy)
         http_session.proxies.update({'http': proxy, 'https': proxy})
-    report = validate(source, format=_format, schema=schema, http_session=http_session, **options)
+        frictionless_context['http_session'] = http_session
```

Comment:
The `http_session` can be also used without `use_proxy` being True, so this needs to be one less level of indentation